### DISCUSSION TITLE
Invoke-SCCMtasks

### DISCUSSION THREAD

**Discussion by bccgeorge** [PSAppDeployToolkit/PSAppDeployToolkit#616] [Q&A] [answered]
I'm looking for some help with the psadt guide.   There's an Invoke-SCCMtask in the help guide that refers to a parameter called ApplicationManagerPolicyAction.  Is this the application deployment evaluation cycle?

**Comment by luki1412** [comment, marked as answer]
Yes

**Reply by bccgeorge** [reply]
Thank you.  That's a bit of weight off my shoulders
